core/sile.lua

```diff
@@ -251,11 +251,19 @@ local function suggest_luarocks (module)
 
         luarocks --lua-version %s --tree lua_modules install %s
 
-    This will install the LuaRocks to your project, then you need to tell your
-    shell to pass along that info about available LuaRocks paths to SILE. This
-    only needs to be done once in each shell.
-
-        eval $(luarocks --lua-version %s --tree lua_modules path)
+    This will install the LuaRocks to your project Note this takes advantage of
+    the fact that SILE checks for modules in the path `lua_modules` relative to
+    the current project by default. If you use this incantation, you should be
+    done.
+
+    In the event you use a different path to the LuaRocks tree, you must also
+    set environment variables to teach SILE about how to find the tree *before*
+    it runs. This can be aided by asking LuaRocks to output a path variable, but
+    it must also be suplimented to keep SILE's original knowledge of paths. This
+    only needs to be done once in each shell, (obviously substituting
+    lua_modules for your actual path):
+
+        eval $(luarocks --lua-version %s --tree lua_modules path --no-bin --append | sed -e "s/'/';;/")
 
     Thereafter running SILE again should work as expected:
```

Comment:
BTW, is this paragraph necessary? One would hope it _should_ work after doing all this, and we are not bringing more information...

Reply:
Agreed. It originally *did* have new info, then I simplified it and it didn't occur to me that it was no longer adding anything. I'm adding a new CLI argument though, so it's going to be used again in a minute...